packages/pipcook-board-client/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+const { eslint } = require('@ice/spec');
+
+module.exports = eslint;
```

Comment:
EOF

Reply:
```js
module.exports = require('@ice/spec').eslint;
```

Reply:
This is the default file generated by [ice scaffold.](https://ice.alibaba-inc.com/) At this stage I think we can just remain this